### DISCUSSION TITLE
Limit Canvas Size to Banner?

### DISCUSSION THREAD

**Discussion by corbinbalzan** [matteobruni/tsparticles#1202] [Q&A] [answered]
Hi @matteobruni !

Thanks for creating such an awesome library! I really appreciated your work :) 

Is it possible to limit the canvas size to a section of the page instead of the length of the body? I'm looking to have the particles as a background to just my main 'hero' section. 

![Screen Shot 2020-12-30 at 3 02 52 AM](https://user-images.githubusercontent.com/29928189/103338167-84720680-4a4b-11eb-836a-d529918be6e9.png)

I can decrease the height in CSS but that ends up condensing the animation. 

Thanks, 
Corbin

**Comment by corbinbalzan** [comment]
I'm also having some trouble with high GPU usage on Firefox and Chrome. I lowered the FPS to 25. I'm hoping a smaller canvas also means less intensive rendering. Would that make sense?

**Reply by matteobruni** [reply]
A smaller canvas could optimize performance if you use `density` in `particles.number` and distances adapted to that size

**Reply by corbinbalzan** [reply]
Excellent! That seemed to help. I also lowered the speed which allowed me to set the FPS to 20.

**Comment by matteobruni** [comment, marked as answer]
You can customize the `<div>` CSS to have a smaller canvas or place it somewhere you prefer, changing `<canvas>` style is not recommended.

**Reply by corbinbalzan** [reply]
Thank you for the quick response! That ended up working once I set the height before the compile. I was editing the height while in dev tools and which caused unintended behavior.
